addons/mail/models/mail_activity.py

```diff
@@ -49,6 +49,18 @@ def default_get(self, fields):
         ('default', 'Other')], default='default',
         string='Category',
         help='Categories may trigger specific behavior like opening calendar view')
+    mail_template_ids = fields.Many2many('mail.template', string='Mails templates')
+    initial_res_model_id = fields.Many2one('ir.model', 'Initial model', compute="_get_initial_res_model_id", store=False, help='Technical field tokeep trace of the model at the beginning of the edition'
+             ' and not available when managing activities for other models.')
+    res_model_change = fields.Boolean(string="Model has change", help="Technical field to indicate IF THE MODEL HAS CHANGE", default=False, store=False)
```

Comment:
Why are you screaming :pill:  ? I would also update the help to tell it is used to trigger UX-related stuff. Otherwise currently the help does not add much value compared to the field name and string.